pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -261,7 +262,7 @@ func (admitter *VMsAdmitter) applyInstancetypeToVm(vm *v1.VirtualMachine) (*inst
 		return nil, nil, spreadConflicts
 	}
 
-	conflicts := admitter.InstancetypeMethods.ApplyToVmi(k8sfield.NewPath("spec", "template", "spec"), instancetypeSpec, preferenceSpec, &vm.Spec.Template.Spec, &vm.Spec.Template.ObjectMeta)
+	conflicts := instancetypeapply.NewVMIApplier().ApplyToVMI(k8sfield.NewPath("spec", "template", "spec"), instancetypeSpec, preferenceSpec, &vm.Spec.Template.Spec, &vm.Spec.Template.ObjectMeta)
```

Comment:
https://github.com/kubevirt/kubevirt/pull/13700 will handle this, can you revert?